src/test/kotlin/com/terraformation/backend/seedbank/AccessionServiceTest.kt

```diff
@@ -101,13 +101,17 @@ internal class AccessionServiceTest : DatabaseTest(), RunsAsUser {
         AccessionModel(
             id = accessionId,
             facilityId = FacilityId(1),
+            isManualState = true,
```

Comment:
What does `isManualState` represent?

Reply:
Literally: it means the accession's state is manually set by the user, as opposed to the system automatically figuring out the state based on things like whether or not the quantity has been entered.

But in practice: it means this accession was created or updated using the v2 API.

The name made sense when we were first starting to implement the new seeds logic; at that point we hadn't decided to split it off into a completely separate set of API endpoints and we thought that "can the user manually edit the state?" was going to be a setting the client would need to know about. But as things turned out, we should have called this field `apiVersion` or `isV2` or something like that.

Reply:
I haven't bothered changing the name because this field will go away once we've gotten rid of the v1 code.

Reply:
Makes sense, thanks!